src/store/accountStore.js

```diff
@@ -139,7 +139,7 @@ export default {
         account.wallet_new(data.walletName, account.wallet_count, 1)
       }
     },
-    importWallet: (context, data) => {
+    importWallet: async (context, data) => {
```

Comment:
async has impact on `src/views/createWallet/createWallet.js:113`, is this being taken into account?